src/loki/worker.cc

```diff
@@ -142,6 +147,21 @@ void loki_worker_t::parse_costing(Api& api, bool allow_none) {
     }
   }
 
+  // Save the edges for chinese postman graph
+  if (options.has_chinese_polygon()) {
+    const auto chinese_edges = edges_in_rings(options.chinese_polygon(), *reader, costing,
+                                              max_chinese_polygon_length, Purpose::CHINESE);
+    for (const auto& edge_id : chinese_edges) {
+      if (avoid_edges.find(edge_id) != avoid_edges.end()) {
+        continue;
+      }
```

Comment:
before this was done in thor, which seems pointless to carry useless chinese edges over in PBF